test/io/literals/mismatch-read-write.good

```diff
@@ -0,0 +1,2 @@
+mismatch-read-write.chpl:18: In function 'main':
```

Comment:
I noticed it was odd you were adding a .good file but not a matching .chpl file.  I don't think you want to add this file, it looks like it gets generated as part of the test, so likely varies depending on where it is run.

To keep other folks from tripping over it as well, you'll want to add a .gitignore to that directory and list this file in the .gitignore.